src/Catalog/Monitoring/Providers/PackageTimestampMetadataResourceV2Provider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
```

Comment:
Duplicate of [src/Catalog/Monitoring/PackageTimestampMetadataResourceV2Provider.cs](https://github.com/NuGet/NuGet.Services.Metadata/pull/151/files#diff-203dac1e9529a031dea1c264ae8ccfe4)?